### ISSUE TITLE
Image Generation Reward Graphs for FLUX LoRA fine-tuning (8 GPUs)

### ISSUE THREAD

**Issue by jaylee2000** [XueZeyue/DanceGRPO#73]
Hello, can you share the reward graphs for FLUX LoRA fine-tuning?
I have seen the graphs for full fine-tuning on 16 and 8 GPUs, but couldn't find the graph for LoRA fine-tuning.

When I ran the scripts, LoRA fine-tuning seems to be slightly slower initiallly, and later catches up but still remains slightly lower.
It seems about right, but I wanted to verify if it's consistent with your results.

<img width="744" height="361" alt="Image" src="https://github.com/user-attachments/assets/025649a9-f41d-47e3-94da-b3bd1f927381" />

**Comment by XueZeyue** [comment]
I'm sorry I forgot to save the reward curve for LoRAs, but I remember that the rewards of LoRAs will be slightly lower than those of full-finetuning.
